### PR TITLE
Create and handle manual broadcasts for events

### PR DESCRIPTION
**Pull request by xmedr**
## Overview

This branch gives metro the ability to manually publish broadcast links for events, in the event that Legistar/Granicus experiences an outage. This ideally only allows one manual link to be active at a time, but there are measures/warnings in place to account for more than one.

By indicating that the presence of a manual broadcast means a meeting is current and ongoing, we're leveraging existing logic that creates and validates a watch live link, and shows current meetings on the homepage.

As seen in the demo below, each event has a button that allows logged in users to create a manual broadcast. Upon creation, an alert for logged in users will be displayed showing any manually created broadcasts. This acts as a reminder to unpublish when the meeting is over. The same button on the same event detail page will unpublish the broadcast, whereas the same button on a different event will be disabled and let the user know to unpublish the previous link before publishing a new one.

There is also a temporary success message that will be displayed upon broadcast creation or deletion.

- Connects #1018

### Demo

https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/3b5d2e34-cb85-4e68-b085-ae1b907e75eb

### Notes

The temporary messages show up separated from the regular alerts to differentiate them, and when the bootstrap 5 changes come in, we can easily make them dismissable.

## Testing Instructions
** Note that since this change requires a database migration, the review app will not work. This will have to be tested locally
 * Head to an event detail page
   * Confirm that the "publish manual live link" button does not appear
 * Log in and go back to an event detail page
   * Confirm that the publish link is now available and shows an appropriate message
 * Create a manual broadcast
   * Confirm that the language surrounding the "publish" button changes, and that an alert visible only to logged in users appears detailing the event you've just manually marked as live.
   * Confirm that an alert-style success message appears underneath the regular alerts, and that it disappears upon refreshing or leaving the page.
 * Head to the index
   * If there are no actual meetings currently happening, confirm that the current meeting is now the one you've manually marked as live.
 * Head to a different meeting's detail page
   * Confirm that the language surrounding the publish link warns of another meeting that is manually live, and that the link itself is disabled.
 * Click the link available to go back to the manually live event, and unpublish it
   * Confirm that the index no longer lists it as current, and the alert with the reminder is gone.
   * Confirm that another success message appears to say that the event has been unpublished.

### CODE REVIEW

**Comment by hancush** [comment]
@xmedr Want to have @antidipyramid take a look at this?

**Comment by xmedr** [comment]
sounds good!

**Review by antidipyramid — CHANGES_REQUESTED** [review]
Great job on this one, @xmedr. Just one question inline.

What do you think about deploying to staging and testing the listen live links out during the next board/committee meeting?

File: lametro/templates/base.html [hunk 1]
```diff
@@ -118,6 +118,37 @@
         {% endfor %}
     {% endif %}
 
+    {% get_events_with_manual_broadcasts as manual_events %}
```
[the remark this answers is not in the document]
Comment (xmedr) [reply lametro/templates/base.html:121]: makes sense!

**Review by antidipyramid — APPROVED** [review]

**Pull request merged**